django/db/backends/mysql/validation.py

```diff
@@ -26,6 +27,29 @@ def _check_sql_mode(self, **kwargs):
             )]
         return []
 
+    def _check_tx_isolation_level(self, **kwargs):
+        if 'isolation_level' in self.connection.settings_dict['OPTIONS']:
+            # User explicitly selected isolation level, trust them.
+            return []
+        with self.connection.cursor() as cursor:
+            cursor.execute("SELECT @@session.tx_isolation")
+            tx_isolation = cursor.fetchone()[0]
+        if tx_isolation == 'REPEATABLE-READ':
+            return [checks.Warning(
+                "Transaction Isolation Level for database connection '%s' is '%s'" % (
```

Comment:
I think it's "Transaction isolation level" i.e. no capitalization of every letter